Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -647,7 +661,19 @@ extension NSManagedObjectContext: MessageDatabaseSession {
         return dto
     }
 
-    func saveMessage(payload: MessagePayload, for cid: ChannelId?, syncOwnReactions: Bool = true) throws -> MessageDTO? {
+    func saveMessages(messagesPayload: MessageListPayload, for cid: ChannelId?, syncOwnReactions: Bool = true) -> [MessageDTO] {
+        let cache = messagesPayload.getPayloadToModelIdMappings(context: self)
+        return messagesPayload.messages.compactMap {
+            try? saveMessage(payload: $0, for: cid, syncOwnReactions: syncOwnReactions, cache: cache)
```

Comment:
(Applies to multiple places) Thrown errors are just ignored now, they should at least be logged

Reply:
There is already a perf test for it. Line 47 - test_measureBigPayload_recursivelyGetAllIds